### DISCUSSION TITLE
Install in a workstation

### DISCUSSION THREAD

**Discussion by wangzhaohao** [idaholab/moose#24362] [Q&A Getting Started] [answered]
Hello, although i install many times in my pc, i meet a problem in a offline workstation. I did the flowing steps.
1. creat a same with pc username.
2. install in pc(online)
3. copy the mambaconfig3 files into the workstation
4. make the test
there is no problem in make, but when i test it fail.
![f46ae040b8fecfb114435b4251994e7](https://github.com/idaholab/moose/assets/51054734/00b225fa-9ef8-41d7-a17c-41c3ffe8a2f8)
Is due to my ```gcc``` is too old?

**Comment by GiudGiud** [comment]
What's your version of gcc?

**Reply by GiudGiud** [reply]
oh copying over the folder wont work

**Reply by GiudGiud** [reply]
please follow these instructions instead
https://mooseframework.inl.gov/getting_started/installation/offline_installation.html

**Reply by wangzhaohao** [reply, marked as answer]
> oh copying over the folder wont work

 the copy is work, i solve this problem because my ```GNULIBC``` is 2.17, but moose need is at least 2.22. i update and it works
